contracts/interfaces/IRiskManagerV2.sol

```diff
@@ -0,0 +1,33 @@
+// ██████████████     ▐████▌     ██████████████
```

Comment:
Let's drop it if it stays empty. And I believe it does based on the previous comment.